virttest/qemu_devices/qcontainer.py

```diff
@@ -1328,6 +1328,9 @@ def machine_s390_virtio(cmd=False):
             bus = (qdevices.QNoAddrCustomBus('bus', [['addr'], [64]],
                                              'virtio-blk-ccw', 'virtio-bus',
                                              'virtio-blk-ccw'),
+                   qdevices.QNoAddrCustomBus('bus', [['addr'], [64]],
```

Comment:
I am not sure it's a correct definition, could you give some references of the length of `virtual-css`?